### ISSUE TITLE
Provide a function to set options

### ISSUE THREAD

**Issue by nevf** [kutlugsahin/smooth-dnd#13]
In a responsive Web app you may want to change the SmoothDND ```options``` on the fly to suite landscape vs portrait etc. Could you please add a function to do this - thanks.

**Comment by kutlugsahin** [comment]
hey @nevf 
You can dispose and recreate it with the new set of options.

**Comment by nevf** [comment]
@kutlugsahin true, however if there are hundreds of elements _(which in my app there is)_ then unwrapping and rewrapping etc. is quite a waste of time and resources, when all that is needed is to change the options. 

All the more so on low powered devices like smartphones where responsive changes are more prevalent.

**Comment by kutlugsahin** [comment]
@nevf. This is true indeed in terms of performance. Lets put this in todo list. Cheers!

**Comment by nevf** [comment]
@kutlugsahin +1 Thanks.

**Comment by xe4me** [comment]
@nevf did you find an easy way to do this ?

**Comment by nevf** [comment]
@xe4me No I didn't. From memory this is an issue which can only be addressed by SmoothDnD which unfortunately seems to be abandoned. I've since written my own drag and drop module which I plan to use in future.
